src/main/java/net/sf/jabref/gui/exporter/ExportFileFilter.java

```diff
@@ -13,26 +13,28 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-package net.sf.jabref.exporter;
+package net.sf.jabref.gui.exporter;
```

Comment:
`import javax.swing.filechooser.FileFilter;`

Had to restructure ExportFormat a bit because of that darn `FileFilter`...